### PR TITLE
fix(elasticBeanstalkEnv): add connections with alb, elb, ec2 and sqs services

### PR DESCRIPTION
**Pull request by hjaraujof**
## Issue tracker links

[CG-334](https://autoclouddev.atlassian.net/browse/CG-334?atlOrigin=eyJpIjoiNjY1ZmYzOGY5YzQxNDJiZWI1NDZlZDc5MWI4NjhiNzciLCJwIjoiaiJ9)

## Dependencies

Depends on #15

### CODE REVIEW

File: src/services/elasticBeanstalkApplication/schema.graphql [hunk 1]
```diff
@@ -5,8 +5,8 @@ type awsElasticBeanstalkApp @key(fields: "arn") {
   region: String @search(by: [hash, regexp])
   name: String! @search(by: [hash, regexp])
   description: String @search(by: [hash, regexp])
-  elasticBeanstalkEnv: [awsElasticBeanstalkEnv] #change to plural
+  iamServiceRole: String @search(by: [hash, regexp])
+  elasticBeanstalkEnvs: [awsElasticBeanstalkEnv]
```
Comment (tyler-dunkel) [inline src/services/elasticBeanstalkApplication/schema.graphql:9]: does this need a has inverse directive?

File: src/services/elasticBeanstalkEnvironment/connections.ts [hunk 2]
```diff
@@ -50,6 +72,139 @@ export default ({
...
+          resourceType: services.ec2Instance,
+          relation: 'child',
+          field: 'ec2Instances',
+        })
+      }
+    }
+  }
+
+  /**
+   * Find Load Balancers(ALB)
```
Comment (tyler-dunkel) [inline src/services/elasticBeanstalkEnvironment/connections.ts:129]: should be (elb)

File: src/services/ec2/schema.graphql [hunk 3]
```diff
@@ -108,15 +108,15 @@ type awsEc2 implements awsBaseService @key(fields: "arn") {
   alb: [awsAlb] @hasInverse(field: ec2Instance)
   asg: [awsAsg] @hasInverse(field: ec2Instance)
   ebs: [awsEbs] @hasInverse(field: ec2Instance)
+  ecsContainer: [awsEcsContainer] @hasInverse(field: ec2Instances)
   eip: [awsEip] @hasInverse(field: ec2Instance)
+  eksCluster: [awsEksCluster] @hasInverse(field: ec2Instances)
+  elasticBeanstalkEnvs: [awsElasticBeanstalkEnv] @hasInverse(field: ec2Instances)
```
[the remark this answers is not in the document]
Comment (tyler-dunkel) [reply src/services/ec2/schema.graphql:114]: Also same question for the above

[the remark this answers is not in the document]
Comment (hjaraujof) [reply src/services/ec2/schema.graphql:114]: 1. An elasticBeanstalkApp can have multiple elasticBeanstalkEnv available to be able to switch between them(versions)
2. No, I'll fix the naming.

**Comment by tyler-dunkel** [comment]
Looks good just needs conflicts resolved.

File: src/utils/index.ts [hunk 4]
```diff
@@ -143,3 +143,7 @@ export const checkAndMergeConnections = (
   }
   return connectionsToMerge
 }
+
+export const caseInsensitiveIncludes = (arr: string[], s1: string): boolean =>
```
Comment (tyler-dunkel) [inline src/utils/index.ts:147]: Do we want to use the built in `includes` function here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

**Review by tyler-dunkel — APPROVED** [review]

**Pull request merged**